### PR TITLE
cmsn-noble

### PR DESCRIPTION
**Pull request by donavanbecker**
cmsn-noble

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: npm-publish
 on:
   push:
     branches:
-      - master # Change this to your default branch
```
Comment (rzr) [inline .github/workflows/npm-publish.yml:5]: shouldn't this be preserved ?

File: .gitignore [hunk 2]
```diff
@@ -18,3 +18,4 @@ npm-debug.log
 lib/mac/native
 
 .vscode/
+package-lock.json
```
Comment (rzr) [inline .gitignore:21]: shouldn't this be preserved ?

File: .github/workflows/npm-publish.yml (hunk 1 above)
Comment (donavanbecker) [reply .github/workflows/npm-publish.yml:5]: Yes

File: .gitignore (hunk 2 above)
Comment (donavanbecker) [reply .gitignore:21]: Yes

**Comment by rzr** [comment]
well I wont work over it but I would appreciate if you could isolate some change in smaller PR then this can be merged one after one

**Pull request closed without merging**